tasks/copy_to_archive_adapter/integration_test_copy_to_archive_adapter.py

```diff
@@ -0,0 +1,93 @@
+import os
+import time
+import unittest
+import uuid
+from unittest.mock import Mock
+
+import boto3
+
+import copy_to_archive_adapter
+
+
+class TestCopyToArchiveAdapter(unittest.TestCase):
+    def test_copy_to_archive_adapter_happy_path(self):
+
+        orca_recovery_bucket = os.environ["ORCA_RECOVERY_BUCKET"]
+        granule_id = uuid.uuid4().__str__()
+        provider_id = uuid.uuid4().__str__()
+        createdAt_time = int((time.time() + 5) * 1000)
+        collection_shortname = uuid.uuid4().__str__()
+        collection_version = uuid.uuid4().__str__()
+        cumulus_bucket_name = "orca-sandbox-s3-provider"
+        execution_id = uuid.uuid4().__str__()
+        key_name = "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"
```

Comment:
Does Cumulus use the same AWS account as us? If not, `key_name` and `cumulus_bucket_name` should be os.environ as well.

Reply:
Might be safer to make them configurable anyway, but I could go either way.

Reply:
different account for sure. 
Changed to making them env variables.